### ISSUE TITLE
Lesson 3 - inconsistent working folder location 

### ISSUE THREAD

**Issue by cszhe** [swcarpentry/git-novice#529]
In lesson 3, the folder `planets` seem to be created directly under user's home directory, as indicated in _Places to Create Git Repositories_ section, as below:

```
$ cd             # return to home directory
$ cd planets     # go into planets directory, which is already a Git repository
...
```
However, as described in _Setup_, this should be created under `Desktop` folder. To be consistent with other lessons, I think this should be corrected.

**Comment by munkm** [comment]
Hi @cszhe, great point! Thank you! Looking at this lesson, I also think your point extends to the beginning of this lesson. It's slightly ambiguous when the first step of the lesson is to `mkdir planets` without reiterating that we're supposed to be at `~/Desktop/`. Would you be interested in submitting a PR to modify this lesson to include one or both of: 

- [ ] a modification of the *Places to Create Git Repositories* exercise to move into `Desktop/planets/` from the home directory rather than `planets/`
- [ ] adding a clarification at the beginning of the lesson to reiterate that `mkdir planets` is executed in the Desktop folder and not the home directory. 

?

**Comment by cszhe** [comment]
Hi @munkm , Thanks for your recognition, I'll submit the PR before this weekend.

**Comment by iglpdc** [comment]
Closed by #530
